streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamPrint.java

```diff
@@ -20,64 +20,28 @@
 import org.apache.kafka.streams.kstream.PrintForeachAction;
 import org.apache.kafka.streams.processor.Processor;
 import org.apache.kafka.streams.processor.AbstractProcessor;
-import org.apache.kafka.streams.processor.ProcessorContext;
 import org.apache.kafka.streams.processor.ProcessorSupplier;
-import org.apache.kafka.common.serialization.Deserializer;
-import org.apache.kafka.common.serialization.Serde;
 
 public class KStreamPrint<K, V> implements ProcessorSupplier<K, V> {
 
-    private final Serde<?> keySerde;
-    private final Serde<?> valueSerde;
     private final ForeachAction<K, V> action;
     
-    public KStreamPrint(final ForeachAction<K, V> action, final Serde<?> keySerde, final Serde<?> valueSerde) {
+    public KStreamPrint(final ForeachAction<K, V> action) {
```

Comment:
Guess, we need to keep the serdes for now to be backward compatible until we really remove all deprecated methods...